### PR TITLE
Upgrade filebeat. Update ES configuration

### PR DESCRIPTION
**Pull request by maria**
- Upgrade filebeat to latest version
	- Update ES configuration
	- Update how we generate filebeat.yml

### CODE REVIEW

**Comment by adragomir** [comment]
:metal:

File: Dockerfile [hunk 1]
```diff
@@ -4,13 +4,11 @@ ENV \
...
-  BEATS_VERSION=1.2.1
+  BEATS_VERSION=5.2.2 \
+  ELASTICSEARCH_INDEX=filebeat
 
 # install dependencies
-RUN apk update \
-    && apk add make git go curl
-
-RUN apk add bash
+RUN apk update && apk add make git go curl libc-dev bash
```
Comment (florinbroasca) [inline Dockerfile]: `apk --no-cache add` no need for `apk update`

File: Dockerfile [hunk 2]
```diff
@@ -26,14 +24,11 @@ RUN echo "Building filebeat" \
...
-RUN echo "Get tools" \
-    && curl -k -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 -o /usr/local/bin/jq \
-    && chmod +x /usr/local/bin/jq
-
+COPY filebeat.yml /etc/filebeat/filebeat.yml
+COPY filebeat.template.json /etc/filebeat/filebeat.template.json
 COPY docker-entrypoint.sh /docker-entrypoint.sh
-COPY filebeat.template.yml /etc/filebeat.template.yml
-COPY filebeat_index.json /etc/filebeat_index.json
+RUN chmod +x /docker-entrypoint.sh
```
Comment (florinbroasca) [inline Dockerfile]: Make `docker-entrypoint.sh` executable in repo?

Comment (maria) [reply Dockerfile]: I think is best to ensure that the file is always executable.

File: Dockerfile [hunk 3]
```diff
@@ -26,14 +24,11 @@ RUN echo "Building filebeat" \
     && GOPATH=/tmp/go-src GOOS=linux make \
     && mv /tmp/go-src/src/github.com/elastic/beats/filebeat/filebeat /usr/local/bin/filebeat
 
-RUN echo "Get tools" \
-    && curl -k -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 -o /usr/local/bin/jq \
-    && chmod +x /usr/local/bin/jq
-
+COPY filebeat.yml /etc/filebeat/filebeat.yml
+COPY filebeat.template.json /etc/filebeat/filebeat.template.json
```
Comment (florinbroasca) [inline Dockerfile]: `COPY filebeat.yml filebeat.template.json /etc/filebeat/`

File: README.md [hunk 4]
```diff
@@ -1,26 +1,22 @@
 # Docker Filebeat image
-This image contains filebeat so you can send logs from a server to Logstash or Elasticsearch.
-The default configuration will send logs to a specific Logstash instance, so it's expecting some environment variables to know where is the Logstash instance.
+This image contains filebeat so you can send logs from a server Elasticsearch.
```
Comment (florinbroasca) [inline README.md]: `to` Elasticsearch?

**Comment by florinbroasca** [comment]
See comments. 🚀.

**Comment by maria** [comment]
@florinbroasca I implemented your review. thanks

**Comment by maria** [comment]
@adragomir @florinbroasca please don't forget to give me push access or rebase via the merge button, because I don't have rights.

**Pull request merged**